reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Variables.md

```diff
@@ -105,14 +105,37 @@ $Splat = @{ Name = "Win*"; Include = "WinRM" }
 Invoke-Command -Session $s -ScriptBlock { Get-Service @Using:Splat }
 ```
 
-## Using local variables in PowerShell 2.0
+### Other situations where the 'Using' scope modifier is needed
+
+For any script or command that executes out of session, you need the `Using`
+scope modifier to embed variable values from the calling session scope, so that
+out of session code can access them. The `Using` scope modifier is supported in
+the following contexts:
+
+- Remotely executed commands, started with `Invoke-Command` using the
+  **ComputerName** or **Session** parameter (remote session)
+- Background jobs, started with `Start-Job` (out-of-process session)
+- Thread jobs, started via `Start-ThreadJob` (separate thread session)
+
+Depending on the context, embedded variables are copied values of the variables
+in the caller's scope. Out-of-process sessions, such as remote sessions,
+background jobs, and workflows, receive values of variables through the
+PowerShell serialization system.
```

Comment:
Assuming this enumeration is complete: "such as" -> "namely"

Reply:
Changed